docs/upgrade-notes/upgrading-tf-version-1.5.3.md

```diff
@@ -0,0 +1,42 @@
+---
+id: upgrade_tf_version_1.5.3
+title: Upgrade to TF version 1.5.3
+hide_title: false
+---
+
+## Background
+
+Cumulus pins its support to a specific version of Terraform [see: deployment documentation](../deployment/README.md#install-terraform). The reason for only supporting one specific Terraform version at a time is to avoid deployment errors than can be caused by deploying to the same target with different Terraform versions.
```

Comment:
```suggestion
Cumulus pins its support to a specific version of Terraform [see: deployment documentation](../deployment/README.md#install-terraform). The reason for only supporting one specific Terraform version at a time is to avoid deployment errors than can be caused by deploying to the same target with different Terraform versions as a rollback to a previous Terraform version is not possible.
```

Reply:
I'm fairly sure this is true. If you've upgrade your terraform version I don't think you can re-apply with an older version. Just trying to make the consequences explicit.